.changesets/2464.md

```
@@ -0,0 +1,20 @@
+- fix(codegen): Read Prisma model names from the generated models file (#2464)
+  by @ladderschool
+
+GraphQL codegen imported the whole generated Prisma client just to read
+`Prisma.ModelName` — the only thing it takes from the client. With a TypeScript
+client that import fails outright in a CommonJS api, because Node resolves the
+`.ts` file as CommonJS and the generated client is ESM:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)